content/docs/hooks-reference.md

```diff
@@ -63,24 +64,24 @@ function Counter({initialCount}) {
 }
 ```
 
-The "+" and "-" buttons use the functional form, because the updated value is based on the previous value. But the "Reset" button uses the normal form, because it always sets the count back to 0.
+Los botones "+" y "-" usan la forma funcional, porque el valor actualizado se basa en el valor anterior. Pero el botón "Restablecer" usa la forma normal, porque siempre vuelve a establecer la cuenta en 0.
 
-> Note
+> Nota
 >
-> Unlike the `setState` method found in class components, `useState` does not automatically merge update objects. You can replicate this behavior by combining the function updater form with object spread syntax:
+> A diferencia del método `setState` que se encuentra en los componentes de la clase,`useState` no combina automáticamente los objetos. Puede replicar este comportamiento combinando la función de actualizador de función con la sintaxis spread de objetos:
 >
 > ```js
 > setState(prevState => {
->   // Object.assign would also work
+>   // Object.assign también funcionaría
 >   return {...prevState, ...updatedValues};
 > });
 > ```
 >
-> Another option is `useReducer`, which is more suited for managing state objects that contain multiple sub-values.
+> Otra opción es `useReducer`, que es más adecuada para administrar estado de objetos que contienen múltiples subvalores.
 
-#### Lazy initialization
+#### Inicialización perezosa
```

Comment:
What do you think about `Inicialización Lenta`?

Reply:
I dont know, `lenta` for me is like it is doing the initialization in a slow way and at some point it will be done

Reply:
sounds better, i will change it now

Reply:
This is a hard one to translate, what about `Inicialización diferida`? That gets closer to what `Lazy Initialization` achieves.

Reply:
lets do a vote what do u think?
+1 on `diferida`